src/mlpack/bindings/julia/CMakeLists.txt

```
@@ -63,8 +63,61 @@ if (BUILD_JULIA_BINDINGS)
       "${CMAKE_BINARY_DIR}/src/mlpack/bindings/julia/mlpack/src/functions.jl"
       "# This file imports all of the mlpack functions into the mlpack module."
       "\n\n")
+
+  file(WRITE
+      "${CMAKE_BINARY_DIR}/src/mlpack/bindings/julia/mlpack/src/types.jl"
+      "# This file defines all of the mlpack types."
+      "\n\n")
+
+  file(WRITE
+      "${CMAKE_BINARY_DIR}/src/mlpack/bindings/julia/mlpack/src/serialization.jl"
+      "# This file imports all serialization and deserialization functions\n"
+      "# from internal modules.\n"
+      "using Serialization\n"
```

Not a big deal, but you might want to `import Serialization` instead of `using` here, so that you aren't accidentally calling any Serialization-internal functions throughout this file? But again, not a big deal really, either way. :)
```suggestion
      "import Serialization\n"
```

---

Hmm, I guess you're right---I'm always using the qualified names `Serialization.serialization()`, etc., in the rest of the generated code (...which lives in the very faraway file `CMake/julia/AppendSerialization.cmake`), so I think this will work fine.  Thanks! :+1: